lib/election.rb

```diff
@@ -1,2 +1,45 @@
 class Election
+    attr_reader :year,
+                :races
+
+    def initialize(year)
+        @year = year
+        @races = []
+
+    end
+
+    def add_race(race)
+        @races << race
+    end
+
+    def candidates
+        candidate_list =[]
+        @races.each do |race|
+            race.candidates.each do |candidate|
+                candidate_list << candidate
+            end
+        end
+        candidate_list
+    end
+
+    def vote_counts
+        vote_hash = {}
```

Comment:
OOOO! Nice hash building here. Again there might be a better enum to use here, but this works great